README.md

```diff
@@ -1,3 +1,21 @@
-# portgraph
+portgraph
+=========
 
-Data structure library for port graphs.
\ No newline at end of file
+[![build_status][]](https://github.com/CQCL/portgraph/actions)
+[![msrv][]](https://github.com/rust-lang/hashbrown)
```

Comment:
why is there a link to hashbrown

Reply:
I copied their format for the badge, and missed that on the rendered md